base/redactor_systemdata.go

```diff
@@ -27,17 +29,41 @@ func (sd SystemData) Redact() string {
 	return clog.Tag(clog.SystemData, string(sd)).(string)
 }
 
+func sds(interfaceSlice interface{}) SystemDataSlice {
+	valueOf := reflect.ValueOf(interfaceSlice)
+
+	length := valueOf.Len()
+	retVal := make([]SystemData, 0, length)
+	for i := 0; i < length; i++ {
+		retVal = append(retVal, SD(valueOf.Index(i).Interface()).(SystemData))
+	}
+
+	return retVal
+}
+
+func (sdSlice SystemDataSlice) Redact() string {
+	tmp := []byte{}
+	for _, item := range sdSlice {
+		tmp = append(tmp, []byte(item.Redact())...)
+		tmp = append(tmp, ' ')
+	}
+	return "[ " + string(tmp) + "]"
+}
```

Comment:
Evaluate using a common redact for the system/meta/user data.

Reply:
Managed to move a couple functions into common file.